cypress/integration/press.spec.ts

```diff
@@ -1,38 +1,37 @@
 describe("cy.realPress", () => {
-  context("key pressing for google", () => {
-    beforeEach(() => {
-      cy.visit("https://google.com?hl=en");
-    });
-
-    it("registers keypress events using", () => {
-      cy.get("input[name=q]").focus();
-
-      cy.realPress("c");
-      cy.realPress("y");
-      cy.realPress("p");
-      cy.realPress("r");
-      cy.realPress("e");
-      cy.realPress("s");
-      cy.realPress("s");
-
-      cy.get("input[name=q]").should("have.value", "cypress");
-    });
-
-    it("Can fire native Tab focus switch", () => {
-      cy.get("input[name=q]").click();
-      cy.realPress("Tab");
-      cy.get("[aria-label='Search by voice']").should("be.focused");
-    });
-
-    it("Can use Enter for a11y navigation", () => {
-      cy.get("input[name=q]").focus();
-      cy.realType("something");
-      cy.realPress("Tab");
+  it("Can type into an input", () => {
+    cy.intercept('http://presstest.com/', (req) => {
+      const html = document.implementation.createHTMLDocument()
+      html.body.innerHTML = `<input type="text">`
+      req.reply(html.documentElement.innerHTML)
+    })
+    cy.visit('http://presstest.com/')
+    cy.get("input").focus();
+
+    cy.realPress("c");
+    cy.realPress("y");
+    cy.realPress("p");
+    cy.realPress("r");
+    cy.realPress("e");
+    cy.realPress("s");
+    cy.realPress("s");
+
+    cy.get("input").should("have.value", "cypress");
+  });
 
-      cy.get("input[name=q]").should("have.value", "something");
-      cy.realPress("Enter");
-      cy.get("input[name=q]").should("have.value", "");
-    });
```

Comment:
I've removed this one as it was basically testing the same thing as this one:
https://github.com/dmtrKovalenko/cypress-real-events/blob/b072ad288e6d4dfe2e3fb085162334950c695c9e/cypress/integration/press.spec.ts#L86-L91